--- a/third_party/starboard/wpe/shared/platform_configuration/BUILD.gn
+++ b/third_party/starboard/wpe/shared/platform_configuration/BUILD.gn
@@ -56,6 +57,8 @@
 config("platform_configuration") {
   ldflags = [ 
     "-Wl,--wrap=eglGetDisplay"
+    ,"-lEGL"
+    ,"-lGLESv2"
   ]
   cflags = []
   if (has_ocdm == "1") {
@@ -130,6 +130,7 @@
     # Cleanup unused sections
    "-Wl,-gc-sections",
     # "-Wl,--unresolved-symbols=ignore-in-shared-libs",
+    "-Wl,--no-undefined"
   ]
 
   cflags += [
