From eb9ac3cc2ce5ab033d820844d5a42df6064ce049 Mon Sep 17 00:00:00 2001
From: Bram Oosterhuis <mail@bybram.com>
Date: Fri, 24 Jun 2022 22:48:02 +0200
Subject: [PATCH] Add mgmt.h as installable

---
 Makefile.am | 4 ++--
 Makefile.in | 6 +++---
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 82125c4..d341c28 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -67,9 +67,9 @@ plugin_LTLIBRARIES =
 lib_sources = lib/bluetooth.c lib/hci.c lib/sdp.c
 lib_headers = lib/bluetooth.h lib/hci.h lib/hci_lib.h \
 		lib/sco.h lib/l2cap.h lib/sdp.h lib/sdp_lib.h \
-		lib/rfcomm.h lib/bnep.h lib/cmtp.h lib/hidp.h
+		lib/rfcomm.h lib/bnep.h lib/cmtp.h lib/hidp.h lib/mgmt.h
 
-extra_headers = lib/mgmt.h lib/uuid.h lib/a2mp.h lib/amp.h
+extra_headers = lib/uuid.h lib/a2mp.h lib/amp.h
 extra_sources = lib/uuid.c
 
 local_headers = $(foreach file,$(lib_headers), lib/bluetooth/$(notdir $(file)))
diff --git a/Makefile.in b/Makefile.in
index 4f08bf2..4099cd2 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -542,7 +542,7 @@ lib_libbluetooth_internal_la_OBJECTS =  \
 lib_libbluetooth_la_LIBADD =
 am__lib_libbluetooth_la_SOURCES_DIST = lib/bluetooth.h lib/hci.h \
 	lib/hci_lib.h lib/sco.h lib/l2cap.h lib/sdp.h lib/sdp_lib.h \
-	lib/rfcomm.h lib/bnep.h lib/cmtp.h lib/hidp.h lib/bluetooth.c \
+	lib/rfcomm.h lib/bnep.h lib/cmtp.h lib/hidp.h lib/mgmt.h lib/bluetooth.c \
 	lib/hci.c lib/sdp.c
 @LIBRARY_TRUE@am_lib_libbluetooth_la_OBJECTS = $(am__objects_1) \
 @LIBRARY_TRUE@	$(am__objects_3)
@@ -2697,7 +2697,7 @@ DATA = $(conf_DATA) $(dbus_DATA) $(dbussessionbus_DATA) \
 	$(systemdsystemunit_DATA) $(systemduserunit_DATA)
 am__pkginclude_HEADERS_DIST = lib/bluetooth.h lib/hci.h lib/hci_lib.h \
 	lib/sco.h lib/l2cap.h lib/sdp.h lib/sdp_lib.h lib/rfcomm.h \
-	lib/bnep.h lib/cmtp.h lib/hidp.h
+	lib/bnep.h lib/cmtp.h lib/hidp.h lib/mgmt.h
 HEADERS = $(pkginclude_HEADERS)
 am__tagged_files = $(HEADERS) $(SOURCES) $(TAGS_FILES) $(LISP) \
 	config.h.in
@@ -3205,7 +3205,7 @@ plugin_LTLIBRARIES = $(am__append_29) $(am__append_30) \
 lib_sources = lib/bluetooth.c lib/hci.c lib/sdp.c
 lib_headers = lib/bluetooth.h lib/hci.h lib/hci_lib.h \
 		lib/sco.h lib/l2cap.h lib/sdp.h lib/sdp_lib.h \
-		lib/rfcomm.h lib/bnep.h lib/cmtp.h lib/hidp.h
+		lib/rfcomm.h lib/bnep.h lib/cmtp.h lib/hidp.h lib/mgmt.h
 
 extra_headers = lib/mgmt.h lib/uuid.h lib/a2mp.h lib/amp.h
 extra_sources = lib/uuid.c
-- 
2.34.1

