From a82f1db2fca75b8bff2cdff2d1ca7eaea9858727 Mon Sep 17 00:00:00 2001
From: Bram Oosterhuis <dev@bybram.com>
Date: Wed, 28 Feb 2024 15:33:33 +0100
Subject: [PATCH] add soversion to EGL and GLESv2

Signed-off-by: Bram Oosterhuis <dev@bybram.com>
---
 interface/khronos/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/interface/khronos/CMakeLists.txt b/interface/khronos/CMakeLists.txt
index 9ad615b..4761180 100644
--- a/interface/khronos/CMakeLists.txt
+++ b/interface/khronos/CMakeLists.txt
@@ -78,6 +78,9 @@ target_link_libraries(GLESv2 EGL khrn_client vcos)
 target_link_libraries(WFC EGL)
 target_link_libraries(OpenVG EGL)
 
+set_target_properties(EGL PROPERTIES SOVERSION 1)
+set_target_properties(GLESv2 PROPERTIES SOVERSION 2)
+
 install(TARGETS EGL GLESv2 OpenVG WFC khrn_client DESTINATION lib)
 install(TARGETS EGL_static GLESv2_static khrn_static DESTINATION lib)
 
-- 
2.34.1

