From f107dcfd91b1493acd7e351640670102dce5ef31 Mon Sep 17 00:00:00 2001
From: jwanna001 <jeff_wannamaker@cable.comcast.com>
Date: Wed, 21 Jul 2021 10:13:23 -0400
Subject: [PATCH 01/27] core: update dependencies for linux dmabuf protocol

Change-Id: Iabe9bee2698b66b9b363145808e6907d5a3c4412
Signed-off-by: jwanna001 <jeff_wannamaker@cable.comcast.com>
---
 Makefile.am | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Makefile.am b/Makefile.am
index bcfcd80..6ebb3ff 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -164,6 +164,7 @@ libwesteros_compositor_la_include_HEADERS = \
    westeros-version.h \
    westeros-render.h \
    protocol/vpc-client-protocol.h
+libwesteros_compositor_la_DEPENDENCIES =
 libwesteros_compositor_la_includedir = $(includedir)
 libwesteros_compositor_la_CXXFLAGS = $(AM_CXXFLAGS) -I$(srcdir)/protocol
 libwesteros_compositor_la_LDFLAGS= \
@@ -195,12 +196,14 @@ libwesteros_compositor_la_LDFLAGS += -lwesteros_simplebuffer_server
 endif
 
 if ENABLE_WESTEROS_LDBPROTOCOL
+libwesteros_compositor_la_DEPENDENCIES += libwesteros_linux_dmabuf_server.la
 libwesteros_compositor_la_LDFLAGS += -lwesteros_linux_dmabuf_server
 endif
 
 ## --- Renderer: OpenGL -------
 libwesteros_render_gl_la_SOURCES = westeros-render-gl.cpp
 libwesteros_render_gl_la_includedir = $(includedir)
+libwesteros_render_gl_la_DEPENDENCIES =
 if !ENABLE_PLATFORM_EMBEDDED
 libwesteros_render_gl_la_LDFLAGS= $(WSTEGL_LIBS) -lEGL -lGL -lGLEW $(WAYLAND_SERVER_LIBS)
 else
@@ -210,11 +213,13 @@ if ENABLE_WESTEROS_SBPROTOCOL
 libwesteros_render_gl_la_LDFLAGS += -lwesteros_simplebuffer_server
 endif
 if ENABLE_WESTEROS_LDBPROTOCOL
+libwesteros_render_gl_la_DEPENDENCIES += libwesteros_linux_dmabuf_server.la
 libwesteros_render_gl_la_LDFLAGS += -lwesteros_linux_dmabuf_server
 endif
 
 ## --- Renderer: Embedded -------
 libwesteros_render_embedded_la_SOURCES = westeros-render-embedded.cpp
+libwesteros_render_embedded_la_DEPENDENCIES =
 libwesteros_render_embedded_la_includedir = $(includedir)
 if !ENABLE_PLATFORM_EMBEDDED
 libwesteros_render_embedded_la_LDFLAGS= -lEGL -lglut -lGL -lGLEW $(WAYLAND_SERVER_LIBS)
@@ -225,6 +230,7 @@ if ENABLE_WESTEROS_SBPROTOCOL
 libwesteros_render_embedded_la_LDFLAGS += -lwesteros_simplebuffer_server
 endif
 if ENABLE_WESTEROS_LDBPROTOCOL
+libwesteros_render_embedded_la_DEPENDENCIES += libwesteros_linux_dmabuf_server.la
 libwesteros_render_embedded_la_LDFLAGS += -lwesteros_linux_dmabuf_server
 endif
 
-- 
2.25.1

